neuralmonkey/config/utils.py

```diff
@@ -4,11 +4,7 @@
 the configuration file because calling the functions or the class constructors
 directly would be inconvinent or impossible.
 """
-
 from typing import Callable, TypeVar
-
-import tensorflow as tf
-
 from neuralmonkey.logging import warn
 
```

Comment:
To deprecated bych někam přestěhoval a ten soubor smazal. Vždyť to vůbec nesouhlasí s deklarovaným smyslem tohodle modulu v tom docstringu.

Reply:
Ok, ale to uz je spis na separatni issue.